core/global/platform.gd

```diff
@@ -147,7 +154,7 @@ func load(root: Window) -> void:
 func get_handheld_gamepad() -> HandheldGamepad:
 	if not platform:
 		return null
-	return await platform.get_handheld_gamepad()
+	return platform.get_handheld_gamepad()
```

Comment:
Why was this `await` removed? Seems unrelated to all the other changes and maybe the source of the handheld issues you saw?

Reply:
It's not, I removed it after the issues arose. I was cleaning up warnings from the Godot editor to tidy up the PR and it mentioned await was not necessary here as its not a co-routine..